### PR TITLE
Introduced the aerospike test suite

### PR DESCRIPTION
**Pull request by sergeysviridenko**
Hello!

* Type: new feature

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/incubator/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [ ] I wrote some tests for this PR.

Small description of change: new testing unit and aerospike. Follow .travis.yml instruction.

Thanks

### CODE REVIEW

**Comment by Jurigag** [comment]
Any reason why we need to move it to other namespace etc?

**Comment by sergeyklay** [comment]
At least to run all aerospike-related tests separately. This will allow to make the build/tests on Travis faster, without using continuous checking.

File: tests/_support/AerospikeTester.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * @SuppressWarnings(PHPMD)
+*/
+class AerospikeTester extends \Codeception\Actor
+{
+    use _generated\AerospikeTesterActions;
+
+   /**
+    * Define custom actions here
+    */
```
Comment (AsterAI) [inline tests/_support/AerospikeTester.php]: i think this is extra

Comment (sergeyklay) [inline tests/_support/AerospikeTester.php]: Actually this is autogenerated file by Codeception

File: .travis.yml [hunk 2]
```diff
@@ -82,7 +82,8 @@ before_script:
 script:
   - vendor/bin/phpcs
   - vendor/bin/codecept build
-  - vendor/bin/codecept run -v
+  - vendor/bin/codecept run -v tests/unit
+  - '[[ "$PHP_MAJOR" == "5" ]] || bash vendor/bin/codecept run -v tests/aerospike'
```
Comment (sergeyklay) [inline .travis.yml]: bash ?

File: tests/aerospike/Cache/Backend/AerospikeTest.php [hunk 3]
```diff
@@ -138,7 +138,7 @@ public function testShouldDeleteData()
     public function testShouldUseOutputFrontend()
     {
         $time = date('H:i:s');
-
+
```
Comment (sergeyklay) [inline tests/aerospike/Cache/Backend/AerospikeTest.php]: ?

**Pull request merged**